---
bumperbot_detection/LitterPoint[] litter_points
